_posts/2021-02-03-cockpit-237.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Cockpit 237
+author: marusak
+date: 2021-02-03
+tags: cockpit linux
+slug: cockpit-237
+category: release
+summary: Cockpit with restyled updates page
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 237.
+
+## Updates: Restyling as preparation for upcoming new fatures
+
+Updates page went through some design polishing as preparation for new upcoming features as support
+of [tracer](https://tracer.readthedocs.io/en/latest/) and [kpatch](https://github.com/dynup/kpatch).
+
+... screenshot ... (TODO we need to fix at least the page filling and possibly also dropping title panel).
+
+## Shell: SSH connections to remote machines are only opened when necessary
+
+Cockpit remembers all remote hosts of a session and used to immediately connect to all of them
+when starting the next session. Now a SSH connection is only opened when navigating to the remote
+machine for the first time.
```

Comment:
We should give a rationale here: It restricts possible security issues, as any connected host can do *anything* to any other connected host. (With better verbiage, obviously)